downstream/modules/platform/proc-controller-image-pull-policy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="proc_controller-image-pull-policy_{context}"]
+
+= controller-image-pull-policy
+
+
+. Under *Image Pull Policy*, click on the radio button to select *Always*, *Never*, or *IfNotPresent*.
+. Define the *Image Pull Secrets*
```

Comment:
```suggestion
. To display options under *Image Pull Secrets*, click the arrow to the right.
```
This is a suggestion to capture exactly what the user will be doing. You can also find examples of this in the IBM Style Guide in the [UI elements ](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui)section (See Drop-down list in the table)